docs/hiveutil.md

```diff
@@ -22,12 +22,23 @@ bin/hiveutil create-cluster --base-domain=mydomain.example.com --cloud=aws myclu
 
 #### Create Cluster on Azure
 
-Credentials will be read from `~/.azure/osServicePrincipal.json` typically created via the `az login` command. Alternatively you can specify an Azure credentials file with `--creds-file` or the `AZURE_AUTH_LOCATION` environment variable.
+Credentials will be read from either `~/.azure/osServicePrincipal.json` (or the Windows equivalent of a HOME/HOMEDRIVE/etc), the contents of the `AZURE_AUTH_LOCATION` environment variable, or the value provided with the `--creds-file` parameter (in increasing order of preference). The format for the credentials used for installation/uninstallation follows the same format used by the OpenShift installer:
```

Comment:
We do not call out this Windows equivalent for any other platforms. I don't think it is necessary here, either.

Reply:
Azure is the only cloud where we use the homedir.HomeDir() call which introduces this new behavior of not relying on $HOME. This behavior seemed a little surprising to me, so I thought I'd document it somewhere, but perhaps these alternative HOME locations are just how things regularly happen on Windows and wouldn't surprise anyone running binaries on that OS.

Reply:
Also, should we also move the GCP equivalent to use homedir.Homedir()?

Reply:
Yes, we probably should do the same for GCP, and OpenStack.

Reply:
Considering that hiveutil is a development tool, is it safe to assume that someone running hiveutil on Windows is using Powershell? In that case `~` is valid for them as well.